server/routeHandlers_test.go

```diff
@@ -10,183 +10,118 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-// TODO: It will be far easier and cleaner to write these tests when we support a MockDB and JWT authentication in unit tests
-// For now, I will not run tests that will polute users local databases or require authentication,
-// Tests will also not deeply check data that is returned from users local databases.
-// TLDR: These tests aren't great, but at least they aren't breaking the build :^)
-
 var unitTestUserName = "UNIT_TEST_USER"
 var unitTestGameName = "UNIT_TEST_GAME_NAME"
 
-func TestGetGames(t *testing.T) {
-	// Setup
+func TestTotalGamePlayAuth(t *testing.T) {
+	// Inital Setup
 	e := echo.New()
 	setupRoutes(e)
-	req := httptest.NewRequest(http.MethodGet, "/api/games", nil)
-	rec := httptest.NewRecorder()
-	c := e.NewContext(req, rec)
-
-	// Assertions
-	if assert.NoError(t, getGames(c)) {
-		assert.Equal(t, http.StatusOK, rec.Code)
-
-		var recData map[string]interface{}
-		json.Unmarshal([]byte(rec.Body.String()), &recData)
-		// TODO: Once we actually support a MockDB for unit tests, check the validity of our response
-		assert.True(t, true)
-	}
-}
 
-func TestNewGame(t *testing.T) {
-	/*
-		assert.True(t, true)
-		// Setup
-		e := echo.New()
-		setupRoutes(e)
-		var postPayload = `{ "name" : "` + unitTestGameName + `", "creator" : "` + unitTestUserName + `" }`
-		req := httptest.NewRequest(http.MethodPost, "/api/games", strings.NewReader(postPayload))
-		req.Header.Set("Content-Type", "application/json")
-		rec := httptest.NewRecorder()
-		c := e.NewContext(req, rec)
-
-		// Assertions
-		if assert.NoError(t, newGame(c)) {
-			assert.Equal(t, http.StatusOK, rec.Code)
-
-			var recData map[string]interface{}
-			json.Unmarshal([]byte(rec.Body.String()), &recData)
-			// TODO: Once we actually support a MockDB for unit tests, check the validity of our response
-			assert.True(t, true)
-		}
-	*/
-	assert.True(t, true)
-}
+	// Create an joinable game
+	createRec := httptest.NewRecorder()
+	createReq := httptest.NewRequest(http.MethodPost, "/api/games?name=game_name&creator=creator_name", nil)
+	e.ServeHTTP(createRec, createReq)
```

Comment:
Hmm.. this works when I run it locally but fails with the git actions test. Any ideas?